libbeat/scripts/Makefile

```diff
@@ -397,7 +397,7 @@ package:
 
 	echo "Finished packages for ${BEATNAME}"
 
-package-dashboards:
+package-dashboards: package-setup
```

Comment:
Ok. It's not ideal, but we need to write in the docs that the users needs to run `package-setup` first.